DfE.FindInformationAcademiesTrusts/Pages/Search.cshtml.cs

```diff
@@ -35,19 +42,21 @@ public async Task<IActionResult> OnGetAsync()
         }
 
         Trusts = await GetTrustsForKeywords();
+        PageStatus = Trusts.Pagination;
+        PaginationRouteData = new Dictionary<string, string> { { "Keywords", KeyWords } };
         return new PageResult();
     }
 
-    private async Task<IEnumerable<TrustSearchEntry>> GetTrustsForKeywords()
+    private async Task<IPaginatedList<TrustSearchEntry>> GetTrustsForKeywords()
     {
         return !string.IsNullOrEmpty(KeyWords)
-            ? await _trustSearch.SearchAsync(KeyWords)
-            : Array.Empty<TrustSearchEntry>();
+            ? await _trustSearch.SearchAsync(KeyWords, PageNumber)
+            : new PaginatedList<TrustSearchEntry>(Array.Empty<TrustSearchEntry>(), 0, 0, Constants.SearchPageSize);
```

Comment:
We should make a `PaginatedList.Empty<T>()` method to simplify this code

Reply:
This would also enable us to remove the `Constants.SearchPageSize` as we'd only need it in one class